packages/dds/tree/src/test/shared-tree/fuzz/operationTypes.ts

```diff
@@ -25,6 +25,11 @@ export interface UndoRedo {
 	contents: FuzzUndoRedoType;
 }
 
+export interface FuzzSchemaChange {
```

Comment:
nit: seems like `SchemaChange` is a more consistent name, and `SchemaOp` over `FuzzSchemaOp`.

Reply:
Updated names to `SchemaChange` and `SchemaOp`